admin_manual/configuration_server/caching_configuration.rst

```diff
@@ -113,7 +113,7 @@ The Redis PHP module must be version 2.2.6+. If you are running a Linux
 distribution that does not package the supported versions of this module, or 
 does not package Redis at all, see :ref:`install_redis_label`.
 
-On Debian/Ubuntu/Mint install ``redis-server`` and ``php5-redis`` or ``php7.0-redis``. The installer
+On Debian/Ubuntu/Mint install ``redis-server`` and ``php-redis``. The installer
```

Comment:
I just looked it up:

* trusty (14.04): `php5-redis`
* xenial, yakkety, zesty (16.04, 16.10, 17.04): `php-redis`

I'm fine with this change.